### PR TITLE
Automated test for setting default filter for hosts and datastores

### PR DESCRIPTION
**Pull request by nachandr**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 452941b31847b63863dd71e2d61026919bc7fbf3:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/infrastructure/test_set_default_filter.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+@pytest.fixture(scope="module")
+def provider():
+    return setup_a_provider(prov_class="infra", prov_type="rhevm")
+
+
+def test_set_default_filter_host(provider):
+    sel.force_navigate('infrastructure_hosts')
+    list_acc.select('Filters', 'Apply this filter')
+    sec_default_btn = "//div[@id='accordion']/a[contains(normalize-space(text()), 'Set Default')]"
```
Comment (mfalesni) [inline cfme/tests/infrastructure/test_set_default_filter.py]: This should live somewhere outside of the test scope. At least at the module level but preferably somewhere in the models where it makes sense.

**Comment by mfalesni** [comment]
@nachandr Please ,configure your git, it seems you commited with nachandr@localhost.localdomain which is not good :)

**Comment by dajoRH** [comment]
Lint report for commit 94636c97fee1756c0502be2a6bc56a0a4221ff43:

`cfme/tests/infrastructure/test_set_default_filter.py`:
- :red_circle: Line 2:1: **F401** *'time' imported but unused*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit cd4775900b4aaf9190a484619e011473a3ec77c5:

`cfme/tests/infrastructure/test_set_default_filter.py`:
- :red_circle: Line 2:1: **F401** *'time' imported but unused*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit ef033dc9084eb26115457963b69debc1c6c5d343:

`cfme/tests/infrastructure/test_set_default_filter.py`:
- :bangbang: Line 24:5: **E101** *indentation contains mixed spaces and tabs*
- :heavy_exclamation_mark: Line 24:5: **W191** *indentation contains tabs*
- :bangbang: Line 27:4: **E901** *SyntaxError: invalid syntax*
- :bangbang: Line 41:5: **E101** *indentation contains mixed spaces and tabs*
- :heavy_exclamation_mark: Line 41:5: **W191** *indentation contains tabs*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 14303588fc296ae462b457df6c1dd322ea5e5e95:

`cfme/tests/infrastructure/test_set_default_filter.py`:
- :bangbang: Line 23:92: **E712** *comparison to True should be 'if cond is True:' or 'if cond:'*
- :bangbang: Line 24:5: **E101** *indentation contains mixed spaces and tabs*
- :heavy_exclamation_mark: Line 24:5: **W191** *indentation contains tabs*
- :bangbang: Line 27:4: **E901** *SyntaxError: invalid syntax*
- :bangbang: Line 31:15: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 31:17: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 43:5: **E101** *indentation contains mixed spaces and tabs*
- :heavy_exclamation_mark: Line 43:5: **W191** *indentation contains tabs*
- :bangbang: Line 50:15: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 50:17: **E251** *unexpected spaces around keyword / parameter equals*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 68c518dd51e37dd6e12b8e9c0adf9e561f9b6f94:

`cfme/tests/infrastructure/test_set_default_filter.py`:
- :bangbang: Line 23:92: **E712** *comparison to True should be 'if cond is True:' or 'if cond:'*
- :bangbang: Line 24:5: **E101** *indentation contains mixed spaces and tabs*
- :heavy_exclamation_mark: Line 24:5: **W191** *indentation contains tabs*
- :bangbang: Line 27:4: **E901** *SyntaxError: invalid syntax*
- :bangbang: Line 31:15: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 31:17: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 43:5: **E101** *indentation contains mixed spaces and tabs*
- :heavy_exclamation_mark: Line 43:5: **W191** *indentation contains tabs*
- :bangbang: Line 50:15: **E251** *unexpected spaces around keyword / parameter equals*
- :bangbang: Line 50:17: **E251** *unexpected spaces around keyword / parameter equals*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e1c9fa6f1d0b88c5df915300d0ccbc1dec655a51:

`cfme/tests/infrastructure/test_set_default_filter.py`:
- :heavy_exclamation_mark: Line 14:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 18:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 20:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 21:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 22:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 23:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 24:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 25:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 26:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 27:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 31:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 33:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 34:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 35:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 36:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 37:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 41:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 43:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 44:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 45:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 46:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 47:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 48:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 49:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 50:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 54:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 56:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 57:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 58:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 59:1: **W191** *indentation contains tabs*
- :heavy_exclamation_mark: Line 60:1: **W191** *indentation contains tabs*

Please, rectify these issues :smirk: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 293460ac9ffed526e19873154ef1487fca58896a:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit e7b699214d247057061e4117584f1539a4bfe619:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 9fee83e5e245e4be2002294ec8450ad47aca3152:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit f8b006b46bb83d4abe7fd856fa7f3129316ae13c:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by mfalesni** [comment]
I think this is good to go from the code POV, we'll see the results. There are a couple of finishing touches that I will do once this is merged.

**Comment by dajoRH** [comment]
Lint report for commit 80559fb9bb4c4012a90abb95ec40480812ecc29a:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 8217423d15232d130ab582ca96757d9bb7f6b13b:

Everything seems all right :smile: .
*CFME QE Bot*

**Pull request merged**